.github/workflows/build.yml

```diff
@@ -14,13 +14,39 @@ jobs:
     steps:
     - name: Checkout Repository
       uses: actions/checkout@v2
-    
+
     - name: Setup CI environment
-      uses: yetanalytics/actions/setup-env@v0.0.2
+      uses: yetanalytics/actions/setup-env@v0.0.3
 
     - name: Build Bundle
       run: make bundle
 
+    # Build docker for PR, tags
+    - name: Extract metadata (tags, labels) for Docker
+      id: meta
+      uses: docker/metadata-action@v3
+      with:
+        images: yetanalytics/datasim
+        tags: |
+          type=ref,event=branch
+          type=ref,event=pr
+          type=semver,pattern={{version}}
+          type=semver,pattern={{major}}.{{minor}}
```

Comment:
What exactly do these tags do in the GH action?

Reply:
They tell it how to form the docker tags. Branch goes in directly, semver does cascading tags. Look at the tags for `yetanalytics/xapipe` on dockerhub for an example https://hub.docker.com/repository/docker/yetanalytics/xapipe/tags